ansible_events_ui/db/utils/lostream.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import annotations
+
+import logging
+from io import UnsupportedOperation
+from typing import Tuple, Union
+
+from fastapi import Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ansible_events_ui.db.dependency import get_db_session
+
+LOG = logging.getLogger(__name__)
```

Comment:
Minor: 

1. We use `logger` name in the project
2. Using `__name__` as logger name is a common practice, but not very useful in my opinion. The main purpose of logger names is to establish logger hierarchy, so that individual loggers can be configured separately. Since we don't have good logging grouping criteria in the project at the moment, it's acceptable to use `__name__` or project name (e.g. "ansible_events") as a logger name. I used the latter across the project.

Reply:
I find it useful. Even if all logs are written to the same destination, you still can see module level information even if you don't have the `basicConfig` set to explicitly log that information.

Reply:
The logger name and tracking of a module where log were written serve the different purposes. The logger name is used to define logger hierarchy, which is rarely map 1:1 to package and modules hierarchy.